pages/settings/preferences/index.vue

```diff
@@ -78,10 +78,10 @@ const userSettings = useUserSettings()
       {{ $t('settings.preferences.hide_reply_count') }}
     </SettingsToggleItem>
     <SettingsToggleItem
-      :checked="getPreferences(userSettings, 'hideFollowerCount')"
-      @click="togglePreferences('hideFollowerCount')"
+      :checked="getPreferences(userSettings, 'hideFollowingFollowerCount')"
+      @click="togglePreferences('hideFollowingFollowerCount')"
```

Comment:
Thanks! I think we should leave the name of the preference as it was before though because if not, everyone will lose this preference and will need to re-enable it. We could add a backward-compatible scheme, but I think the previous name is good enough here.